source/manual/alerts/expiring-api-tokens-in-signon.html.md

```diff
@@ -34,10 +34,10 @@ for Publishing API expires in X days*. In this example, the API User is
 - Copy the new token and prepare to replace it in [govuk-secrets].
 
 > How to do the last step depends on the application, but it should be
-> something like `rake eyaml:edit[integration,apps]`, depending on the
+> something like `rake eyaml:edit[integration,apps]` within the `puppet_aws` directory, depending on the
```

Comment:
We could also suggest the non 'apps' eyaml as a potential place to look: `rake eyaml:edit[integration]`.

Reply:
tiny thing, maybe include the single quotes required in eyaml (i.e. `rake 'eyaml:edit[integration,apps]'`)